spec.md

```diff
@@ -199,6 +206,7 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 | [dashboard_client](#dashboard-client-object) | object | Contains the data necessary to activate the Dashboard SSO feature for this service. |
 | plan_updateable | boolean | Whether the service supports upgrade/downgrade for some plans. Please note that the misspelling of the attribute `plan_updatable` to `plan_updateable` was done by mistake. We have opted to keep that misspelling instead of fixing it and thus breaking backward compatibility. Defaults to false. |
 | [plans*](#plan-object) | array-of-objects | A list of plans for this service, schema is defined below. MUST contain at least one plan. |
+| deprecated | boolean | Indicates that this service and all of its plans are deprecated and will be removed in the future. See [deprecating services and plans](#deprecating-services-and-plans) for more information. |
```

Comment:
s/will/might/
The future may find a reason to revert from deprecated to not-deprecated.